src/modules/kvs/commit.c

```diff
@@ -1253,6 +1253,7 @@ int commit_mgr_merge_ready_commits (commit_mgr_t *cm)
 {
```

Comment:
maybe it would be clearer to make the commit summary "modules/kvs: don't modify ready queue on error"?